packages/vitest/src/node/reporters/junit.ts

```diff
@@ -205,7 +179,12 @@ export class JUnitReporter implements Reporter {
               if (!error)
                 return
 
-              await this.writeErrorDetails(task, error)
+              const result = await printErrorWrapper(
+                error,
+                this.ctx,
+                this.ctx.getProjectByTaskId(task.id),
+              )
+              await this.baseLog(escapeXML(stripAnsi(result.output.trim())))
```

Comment:
I stripped the color for now, but it would be cool to have diff color if circleci (or other consumers of junit reporter) supports ansi code on their UI.